src/SGFTree.cpp

```diff
@@ -477,6 +477,10 @@ std::string SGFTree::state_to_string(GameState& pstate, int compcolor) {
         }
     }
 
+    auto tc = state->get_timecontrol();
+    header.append("\nC[LeelaZ options: " + cfg_options_str + "\n");
```

Comment:
I don't like hardcoding the program name, there's a PROGRAM_NAME define.

Reply:
Fixed.